src/test.sh

```diff
@@ -3,13 +3,14 @@
 set -u # crash on missing env
 set -e # stop on any error
 
+# Coverage 6: coverage run --data-file=/tmp/.coveragerc …
 export COVERAGE_FILE=/tmp/.coverage
 
+echo "Running style checks"
```

Comment:
I think that running tests is more important than running style checks. When fixing bugs i want to know if my code runs fine first, then i want to know if flake8 agrees with how it looks.